tests/tests/swfs/stage3d/shared/com/adobe/utils/AGALMiniAssembler.as

```diff
@@ -372,7 +372,7 @@ package com.adobe.utils
 								if ( optfound == null )
 								{
 									// todo check that it's a number...
-									//trace( "Warning, unknown sampler option: "+opts[k] );
+									trace( "Warning, unknown sampler option: "+opts[k] );
```

Comment:
Intentional change or leftover from debugging?

Reply:
Intentional - this makes it easier to write agal shaders